File: src/main/java/com/example/librarymanagement/dto/BookPresenceDto.java
package com.example.librarymanagement.dto;

import com.example.librarymanagement.model.enums.Availability;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class BookPresenceDto {
    private Long id;
    private UserResponseDto user;
    private String bookTitle;
    private Long bookAuthorId;
    private Long libraryNameId;
    private String address;
    private Availability availability;
}

File: src/main/java/com/example/librarymanagement/dto/BookPresenceDto.kt
package com.example.librarymanagement.dto

import com.example.librarymanagement.model.enums.Availability

class BookPresenceDto (
    val id: String? = null,
    val user: UserResponseDto ?= null,
    val bookTitle: String,
    val bookAuthorId: String,
    val libraryId: String,
    val availability: Availability,
)
